### DISCUSSION TITLE
How do I reduce spacing around a literal slash in math mode?

### DISCUSSION THREAD

**Discussion by tavianator** [typst/typst#714] [Q&A]
I'd like to write $\mathbb{Z} / n \mathbb{Z}$.  Writing `$ZZ \/ n ZZ$` renders as

![Screenshot_20230410_105235](https://user-images.githubusercontent.com/1692591/230926129-bbaf28ee-d771-41fb-867f-13bd74a2a823.png)

which IMO has way too much space around the slash.

**Comment by Enivex** [comment]
```
#let myslash = $#h(0em) slash #h(0em)$

$ZZ myslash n ZZ$
```
Change 0em to your liking
